### DISCUSSION TITLE
Auto toggle Night Light Schedule

### DISCUSSION THREAD

**Discussion by xsm2** [Codectory/AutoActions#8] [Ideas]
If you have ever used Windows 10 Night Light you should know that it doesn't work nice with HDR, as the filter it's too strong with SDR night light values.

I would love an option that lets disables the Night Light Schedule when HDR is enable and reenables it when its off.

PD: maybe this could be an example of how to do it: https://github.com/jaapbrasser/SharedScripts/tree/master/Set-BlueLight

**Comment by Codectory** [comment]
Good idea, i will look into this.
Just don't have to much time these days. Maybe next week.

Am Di., 2. März 2021 um 16:16 Uhr schrieb Xavier Siguero Mora <
notifications@github.com>:

**Comment by Codectory** [comment]
Will do this in the next release!

**Comment by Codectory** [comment]
The Powershell script doesn't work for me. I found out, that the registry path changed in some Windows Updates, but none of them I found worked. Does the powershell script work on your computer?

**Reply by xsm2** [reply]
No, it doesn't and It's a pity. I will try to search some info but I dont have high hopes.
